src/js/github-api.js

```diff
@@ -23,18 +24,18 @@ export async function isProviderTokenValid(token) {
 
 /**
  * Fetch pull request data from specified parameters.
- * @param { string } token The SSO token generated
+ * @param { User } user The SSO token generated
```

Comment:
This comment description needs to be updated, since the parameter is the user not the SSO token.